6.bulk_Morphology_Elastic_Network/0.split_data/notebooks/0.split_data_regression.ipynb

```diff
@@ -0,0 +1,872 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #34.        f"Shape for the 50% test set: {training_data_set_50.shape};\nShape for the 50% train set: {testing_data_set_50.shape}"
</pre><p>Maybe this is intentional, but by does the size of (train_set + test_set = 84) for 50% whereas the other percentages only add to 8?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>